aten/src/ATen/native/TensorShape.cpp

```diff
@@ -2073,7 +2073,7 @@ Tensor flatten(const Tensor& self, DimnameList dims, Dimname out_dim) {
 }
 
 Tensor ravel(const Tensor& self) {
-  return self.reshape(-1);
+  return self.contiguous().view(-1);
```

Comment:
Could you add a comment mentioning that `contiguous` is no-op if self is already contiguous.

Reply:
Yep, added. Thanks!